implementation/src/main/java/io/smallrye/restclient/RestClientBuilderImpl.java

```diff
@@ -133,7 +133,7 @@ public RestClientBuilder baseUrl(URL url) {
                 .proxyBuilder(aClass)
                 .classloader(classLoader)
                 .defaultConsumes(MediaType.TEXT_PLAIN)
```

Comment:
Should the default `@Consume` also change?

Reply:
Not sure here, is the default consumes really needed as it produces a Content-type: */* ? Perhaps its better not to send it at all I mean.

Reply:
Ah, you have a point. Maybe remove that line and see if the TCK passes

Reply:
Didn't pass it seems.

 According to RFC 7231 

> If a Content-Type header field is not present, the recipient MAY either assume a media type of "application/octet-stream ([RFC2046], Section 4.5.1) or examine the data to determine its type." 

Should I try `MediaType.APPLICATION_OCTET_STREAM` for the `defaultConsumes`?

Reply:
Sure

Reply:
Seems to be in better shape now!